_layouts/aggregator.html

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+parent: Aggregators
+---
+
+<h1>
+  {{ page.title }}
+</h1>
+<h2>
+  {{ page.description }}
+</h2>
+
+<p>
+  {% if page.supported_engines.size == 1 %}
```

Comment:
Probably we can remove this, if an aggregator only supports one engine, that's not an aggregator, that's a proxy.